Editor/PipeGenerator.cs

```diff
@@ -21,6 +21,7 @@ public class PipeGenerator : MonoBehaviour
         public bool HasCaps;
         public float CapThickness = 1;
         public float CapRadius = 1.3f;
+        public float CapOffset = .25f;
```

Comment:
I think it should be 0 by default, wanting it to be offset is pretty rare after all.

Reply:
Yeah, I realized this one very late last night.

I'd argue that if you want end caps (a.k.a. flanges), you do want them offset (you need access from both sides in order to put the bolts in -- yes, I spent a fair bit of time working on industrial piping systems at one point), unless what you are connecting to is an already stubbed flange on your geometry. In any case, whatever default is used, the user will almost certainly want to change it.